src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingJointJudicialSeparationContent.java

```diff
@@ -79,7 +90,10 @@ public Map<String, Object> apply(final CaseData caseData,
             .town(town)
             .build();
 
-        if (!WELSH.equals(applicantLanguagePreference)) {
+        if (WELSH.equals(applicantLanguagePreference)) {
+            getWelshTemplateContent(templateContent, caseData);
+            ctscContactDetails.setPoBox("Blwch Post 13226");
```

Comment:
how come only post box change and rest of the details stay sam for welsh. Also this not be set like this (hardcoded) it should be in config

Reply:
If we use `DocmosisCommonContent.getBasicDocmosisTemplateContent(LanguagePre...)` it takes care of header and footer values based on language 👍

Reply:
Can I change the welsh email in this method to the contactdivorce@justice.gov.uk?

Reply:
It's already updated as part of NFDIV-3200. That ticket is in QA in progress so may be leave the values bit for now as once that PR is merged you could simply merge master? 🤔